src/features/PeopleList/styled.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const Content = styled.section`
+    max-width: 1368px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Title = styled.h1`
+    font-size: 36px;
+    line-height: 120%;
+    margin: 0;
+    padding: 56px 16px 24px 16px;
```

Comment:
Instead of removing default margin and setting an exact padding you could just set margin like so: 

`margin: 56px 16px 24px;` 

Also last 16px is not needed because if you specify just 3 values, the second one will be applied on both sides.